integration/yarn/src/test/java/alluxio/yarn/ClientTest.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.yarn;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.doReturn;
+
+import alluxio.Configuration;
+import alluxio.Constants;
+import alluxio.PropertyKey;
+import alluxio.exception.ExceptionMessage;
+
+import org.apache.hadoop.yarn.api.protocolrecords.GetNewApplicationResponse;
+import org.apache.hadoop.yarn.api.records.Resource;
+import org.apache.hadoop.yarn.client.api.YarnClient;
+import org.apache.hadoop.yarn.client.api.YarnClientApplication;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+/**
+ * Unit tests for {@link Client}.
+ * TODO(xuan gong): ALLUXIO-1503: add more unit test for alluxio.yarn.Client
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Client.class, YarnClient.class})
+public final class ClientTest {
+
+  private YarnClient mYarnClient;
+
+  @Rule
+  public ExpectedException mThrown = ExpectedException.none();
+
+  @Before
+  public void before() {
+    // Mock Yarn client
+    PowerMockito.mockStatic(YarnClient.class);
+    mYarnClient = (YarnClient) Mockito.mock(YarnClient.class);
+    Mockito.when(YarnClient.createYarnClient()).thenReturn(mYarnClient);
+
+    Configuration.set(PropertyKey.INTEGRATION_MASTER_RESOURCE_MEM, "2048.00MB");
```

Comment:
`Configuration` in Alluxio is a singleton. Setting this through the test will have side-effect across tests.
@aaudiber , what is the current recommended way to clean this up in unit tests (rather than integration tests)?